src/js/components/StarRating/StarRating.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { Star, StarOutline } from 'grommet-icons';
+import { RadioButtonGroup } from '../RadioButtonGroup';
+
+export const StarRating = ({
+  fillColor,
+  name,
+  label,
+  scale = 5,
+  value,
+  onChange,
+  outlineColor,
+  ...rest
+}) => {
+  const [rating, setRating] = useState(value);
+
+  const options = [];
+  for (let i = 0; i < scale; i += 1) {
+    options.push(i);
+  }
+
+  return (
+    <RadioButtonGroup
```

Comment:
I noticed some odd behavior when using voice over and trying to navigate with my keyboard

- If 5/5 stars were selected I could no longer tab to the RadioButtonGroup component.
- When 4/5 stars were selected voice over announced that 5/5 were selected.
- Using my left and right arrow keys I could select up to 4/5, but was unable to select the last star.